### PR TITLE
feat: Add a list of scrapers dropped due to dedup in report.json (#87)

### PR DESCRIPTION
**Pull request by andys1376**

### CODE REVIEW

**Comment by andys1376** [comment]
That makes sense to have it as part of the cross check reports. I will work
on that.

**Comment by andys1376** [comment]
Example output for 2 locations:

image

**Comment by lazd** [comment]
That sample output is looking great! I'll review the code and try to get this in ASAP

File: src/events/processor/dedupe-locations/index.js [hunk 1]
```diff
@@ -42,6 +45,49 @@ function getCleanPath(scraperFilePath) {
...
+  report.agreements = agreements;
+
+  const strippedUsedLocation = transform.removePrivate({ ...usedLocation });
+  if (!existsInCrosscheckSources(strippedUsedLocation, report.sources)) {
+    // Add to beginning of array so that the used source is always at index 0
+    report.sources.unshift(strippedUsedLocation);
+  }
+  const stippedDroppedLocation = transform.removePrivate({ ...droppedLocation });
+  if (!existsInCrosscheckSources(stippedDroppedLocation, report.sources)) {
+    report.sources.push(stippedDroppedLocation);
```
Comment (lazd) [inline src/events/processor/dedupe-locations/index.js]:
```suggestion
  const strippedDroppedLocation = transform.removePrivate({ ...droppedLocation });
  if (!existsInCrosscheckSources(strippedDroppedLocation, report.sources)) {
    report.sources.push(strippedDroppedLocation);
```

**Comment by lazd** [comment]
Alright, I have to change the cross check report page on the site to work with this new data (and drop those that have no discrepancies), but this is honestly looking great. I'll have to do this tomorrow, however. Thanks for the hard work!

**Comment by lazd** [comment]
@andys1376 this is fantastic work, I went ahead and fixed up the cross check report page and it's so much more useful now:

image

THANK YOU!

**Pull request merged**